sopel/plugins/capabilities.py

```diff
@@ -0,0 +1,477 @@
+"""Capability Requests management for plugins.
+
+.. versionadded:: 8.0
+
+.. important::
+
+    This is all relatively new. Its usage and documentation is for Sopel core
+    development and advanced developers. It is subject to rapid changes
+    between versions without much (or any) warning.
+
+    Do **not** build your plugin based on what is here, you do **not** need to.
+"""
```

Comment:
```suggestion
    Do **not** build your plugin based on what is here, you do **not** need to.

"""
```

This is our usual style when indented blocks occur at the end of a docstring.